modules/ktx/texture_loader_ktx.cpp

```diff
@@ -287,8 +287,6 @@ static Ref<Image> load_from_file_access(Ref<FileAccess> f, Error *r_error) {
 							ktxfmt = KTX_TTF_BC7_RGBA;
 						} else if (RS::get_singleton()->has_os_feature("s3tc")) {
 							ktxfmt = KTX_TTF_BC1_RGB;
-						} else if (RS::get_singleton()->has_os_feature("etc")) {
-							ktxfmt = KTX_TTF_ETC1_RGB;
```

Comment:
Based on https://github.com/godotengine/godot/pull/88325/files#r1489704307, I guess this should also be reintroduced, but with proper ETC2 support if it exists.